Formula/hyperkit.rb

```diff
@@ -0,0 +1,120 @@
+class Hyperkit < Formula
+  desc "Lightweight virtualization hypervisor for MacOS"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20170425",
+    :revision => "a9c368bed6003bee11d2cf646ed1dcf3d350ec8c"
+
+  bottle do
+    root_url "http://dl.bintray.com/markeissler/homebrew/bottles"
+    cellar :any_skip_relocation
+    sha256 "5cfa72e41bad9d812206a9850d7e6e63185ce1bffa0d1718beb5f09734d9bb29" => :sierra
+    sha256 "32162cf81ca23a27f97e0fe0727ecc6f3dcf179716ac77fb9fbc38883e3d114f" => :el_capitan
+    sha256 "0fb4cf0f9f8d81eb1be99d620b454ab14ec8680237f6938db095b032874e821d" => :yosemite
+  end
+
+  head do
+    url "https://github.com/moby/hyperkit.git", :branch => "master"
+  end
+
+  depends_on "opam"
+  depends_on "libev"
+
+  resource "tinycorelinux" do
+    url "https://dl.bintray.com/markeissler/homebrew/hyperkit-kernel/tinycorelinux_8.x.tar.gz"
+    sha256 "560c1d2d3a0f12f9b1200eec57ca5c1d107cf4823d3880e09505fcd9cd39141a"
+  end
+
+  def install
+    ohai "Installing hyperkit dependencies with OPAM. This might take a while..."
+
+    quiet_system <<-CMD.undent
+      export OPAMYES=1
+      opam init
+      eval "$(opam config env)"
+      opam install uri qcow.0.10.0 qcow-tool mirage-block-unix.2.7.0 conf-libev logs fmt mirage-unix prometheus-app
+    CMD
+
+    ohai "Dependencies installed."
+
+    # update the Makefile to set version to X.YYYYmmdd (sha1)
+    if build.head?
+      command = <<-CMD.undent
+        \\cd "#{buildpath}"; \
+        \\git log -1 --pretty=format:"vHEAD.%cd-%h" --date=short "master"
+      CMD
+      version_string = system command.chomp
+      version, sha1 = version_string.split("-", 3).join("").split("-")
+    else
+      # grab version and sha1 from stable resource specs
+      version = stable.specs[:tag][0..-1]
+      sha1 = stable.specs[:revision][0..6]
+    end
+
+    if version.nil? || version.empty? || sha1.nil? || sha1.empty?
+      odie "Couldn't figure out which version we're building!"
+    end
+
+    quiet_system <<-CMD.undent
+      \\sed -i".bak" \
+      -e "s/GIT_VERSION[\ ]*:=.*/GIT_VERSION := '#{version} (#{sha1})'/g" \
+      -e "s/GIT_VERSION_SHA1[\ ]:=.*/GIT_VERSION_SHA1 := '#{sha1}'/g" \
+      "#{buildpath}/Makefile"
+    CMD
+
+    system "make"
+
+    bin.install "build/hyperkit"
```

Comment:
Filing such a request will not make me think that you have guaranteed to me that it will happen.

> Look how many rocks have been thrown just to get your revisions into the formula. The upstream hyperkit team has their own turf to protect

Requests for changes are not rocks, nor are they turf protection.

>The upstream hyperkit team has their own turf to protect and their priorities aren't this formula, it's distribution of hyperkit in the full Docker for Mac package.

A `make install` target is not an outrageous request if they're interested in seeing this software packaged and distributed by downstreams such as Homebrew.